web_src/js/features/repo-issue.js

```diff
@@ -185,6 +186,11 @@ export function initRepoIssueCommentDelete() {
           }
           $conversationHolder.remove();
         }
+        // Check if there is no review content, move the time avatar upward to avoid overlapping the content below.
+        if (!$parentTimelineItem.find('.conversation-holder').length) {
```

Comment:
Here seems not right. In one "group", there could be more than one "items"

![image](https://github.com/go-gitea/gitea/assets/2114189/49167393-f93e-4f99-9c0c-189edcd2707c)

Reply:
It seems so, I will continue to make modifications.

Reply:
updated
By adding another constraint: ensure there is no "comment" in the "group"
![image](https://github.com/go-gitea/gitea/assets/37935145/a2d1ac0f-2cf3-4644-b08f-783927844a58)